### PR TITLE
added playlist and history functionalities

### PR DESCRIPTION
**Pull request by anasm11**
Files to check:
-history-context.js
-playlist-context.js
-History.jsx
-Playlists.jsx
-Playlistdetails.jsx

### CODE REVIEW

File: src/contexts/playlist-context.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+case('DELETE_PLAYLIST'):{
+    return payload
+}
+
+case('DELETE_FROM_PLAYLIST'):{
+    state.map((playlist)=>playlist._id===payload._id?payload:playlist)
+    
+}
+
```
Comment (Adarsh-Patel-Dev) [inline src/contexts/playlist-context.js:21]: there should be default case in switch case.

Comment (Adarsh-Patel-Dev) [reply src/contexts/playlist-context.js:21]: Also try to give proper indentation

File: src/pages/History.jsx [hunk 2]
```diff
@@ -1,9 +1,51 @@
...
+
     return (
         <div>
-            history
+            {historyVideos.map((video)=>{
+
+return <div className="card card-with-badge" key={video._id}>
+  <div className="card-body">
+    <div className="figure-with-badge">
+      {/* <img src="..." /> */}
```
Comment (Adarsh-Patel-Dev) [inline src/pages/History.jsx]: remove comment here
Rest code is good to merge

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by gauravsinhaweb — APPROVED** [review]
Adarsh has covered all the points for improvements, you should work on the UI part and make sure if you are implementing API calls for all the features, ,Good to merge after changes

**Pull request merged**